%YAML:1.0

body_T_cam0: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data:  [0, -1, 0, 0,
           -1, 0, 0, 0,
           0, 0, -1, -0.05,
           0, 0, 0, 1.]